src/main/java/ca/on/oicr/gsi/dimsum/util/reporting/reports/MohTglTrackingReport.java

```diff
@@ -0,0 +1,194 @@
+                                                    x.test().getFullDepthSequencings().stream())
+                                            .map(Sample::getId).collect(Collectors.joining(", "))),
+                            Column.forString("Library Aliquot Name (Library Alias)",
+                                    x -> Stream
+                                            .concat(x.test().getLibraryQualifications().stream(),
+                                                    x.test().getFullDepthSequencings().stream())
+                                            .map(Sample::getName).collect(Collectors.joining(", "))),
+                            Column.forString("Library Qualification Runs",
+                                    x -> x.test().getLibraryQualifications().stream()
+                                            .filter(sample -> sample.getRun() != null)
+                                            .map(runlib -> runlib.getRun().getName())
+                                            .collect(Collectors.joining(", "))),
+                            Column.forString("Library Qualification Pass?",
+                                    x -> CompletedGate.LIBRARY_QUALIFICATION.qualifyTest(x.test()) ? "Yes"
+                                            : null),
+                            Column.forString("Full-Depth Runs",
+                                    x -> x.test().getFullDepthSequencings().stream()
+                                            .map(runlib -> runlib.getRun().getName())
+                                            .collect(Collectors.joining(", "))),
+                            Column.forDecimal("Coverage Required", MohTglTrackingReport::getCoverageRequired),
+                            Column.forDecimal("Coverage Achieved", MohTglTrackingReport::getCoverageAchieved),
+                            Column.forString("MOH Data Released", x -> getCaseMOHDelivered(x.kase())),
+                            Column.forString("Case Status", x -> getCaseStatus(x.kase())))) {
+
+                @Override
+                public List<RowData> getData(CaseService caseService,
+                                             JsonNode parameters) {
+                    Set<String> projectNames = getParameterStringSet(parameters, "projects");
+                    List<CaseFilter> filters = null;
+                    if (projectNames != null) {
+                        filters = projectNames.stream()
+                                .map(project -> new CaseFilter(CaseFilterKey.PROJECT, project))
+                                .toList();
+                    }
+
+                    Map<Long, Assay> assaysById = caseService.getAssaysById();
+
+                    return caseService.getCaseStream(filters)
+                            .flatMap(kase -> kase.getTests().stream()
+                                    .flatMap(test -> test.getFullDepthSequencings().stream()
```

Comment:
What is the point of reporting per full-depth sequencing run? The coverage metrics (which are merged), data release, and case status will all be the same for all full-depth sequencing within the same test.